docs/apv.md

```diff
@@ -134,7 +134,7 @@ Het Wetboek van Strafrecht, specifiek bedoeld voor Tedeapolis, wordt hierbij ing
 
 ### Artikel 9 - Gebruik van voertuigen
 
-1. Je mag als burger geen overheidsvoertuigen besturen, tenzij er een goede roleplay aan vast zit, welke goedgekeurd is door een stafflid.
+1. Je mag als burger geen overheidsvoertuigen besturen, tenzij er in de roleplay een goede reden voor is.
```

Comment:
Ik zou hier wel nog de eventuele goedkeuring van een stafflid in verwerken eerlijk gezegd, dan kunnen ze het niet gooien op "het staat er niet in"

Reply:
Dat was met de oude APV ook niet het geval en dat zou dus betekenen dat je bijvoorbeeld in een achtervolging in nood niet een politievoertuig zou mogen pakken. @musician1011

Reply:
Waarom wordt dit ineens aangepast? @ARJ3N

Reply:
@ARJ3N als je ervan maakt "Je mag als burger geen overheidsvoertuigen besturen, tenzij er in de roleplay een goede reden voor is, of wanneer er toestemming is gegeven door een stafflid." Dan blijven beide opties open toch?

Reply:
Dat is ook prima, maar komt in principe op hetzelfde neer, dus dat maakt mij niet uit. @musician1011

Reply:
Als je dat nog even doet, dan voer ik 'm door. :)

Reply:
Gedaan! :) @musician1011